### PR TITLE
Add long code examples in procedures

### PR DESCRIPTION
**Pull request by cbudz**
Entry for review @redhat-documentation/ccs-style-council

### CODE REVIEW

File: supplementary_style_guide/style_guidelines/code-commands.adoc [hunk 1]
```diff
@@ -10,3 +10,5 @@
 
 [[long-code-examples]]
 == Long code examples in procedures
+
+All code blocks, regardless of length, should be necessary, accurate, and helpful. Code blocks should be as copy-and-paste friendly as possible, with the exception of variables and callouts only. The length of the block is irrelevant, within reason, if the code block follows these guidelines. For more information about code sample suggestions, see IBM Style Guide pp. 198-200.
```
Comment (bergerhoffer) [inline supplementary_style_guide/style_guidelines/code-commands.adoc:14]: I'd remove the last sentence about the IBM style guide. You could really say that to any of these guidelines, plus the page numbers might not be accurate (I have an online version that has no page numbers).

**Review by bergerhoffer — COMMENTED** [review]
Just one suggestion, but otherwise LGTM

File: supplementary_style_guide/style_guidelines/code-commands.adoc (hunk 1 above)
Comment (ndeevy) [inline supplementary_style_guide/style_guidelines/code-commands.adoc:14]:
```suggestion
All code blocks, regardless of length, must be necessary, accurate, and helpful. Code blocks must be as copy-and-paste friendly as possible, with the exception of variables and callouts only. The length of the block is irrelevant, within reason, if the code block follows these guidelines. For more information about code sample suggestions, see IBM Style Guide pp. 198-200.
```

Suggestion to make the 'should's into 'must's for IBM style guidance about being direct.

**Comment by ndeevy** [comment]
@cbudz Review done. LGTM! just one small suggestion.

File: supplementary_style_guide/style_guidelines/code-commands.adoc (hunk 1 above)
Comment (cbudz) [reply supplementary_style_guide/style_guidelines/code-commands.adoc]: Removed

Comment (cbudz) [reply supplementary_style_guide/style_guidelines/code-commands.adoc]: @ndeevy made these updates here

**Review by bergerhoffer — APPROVED** [review]
LGTM now, thanks!

**Comment by cbudz** [comment]
thanks @bergerhoffer merging this.

**Pull request merged**